sidebars.js

```diff
@@ -85,6 +85,17 @@ module.exports = {
         'authorizer-guide/policy',
       ],
     },
+    {
+      'Decision Logs Guide': [
+        'decision-logs-guide/overview',
+        'decision-logs-guide/console',
+        'decision-logs-guide/api',
+        'decision-logs-guide/decision_logs',
```

Comment:
We should probably have the `decision_logs`, `users` and `queries` pages under `api`

Reply:
Done